bitnami/argo-cd/templates/repo-server/deployment.yaml

```diff
@@ -273,9 +309,82 @@ spec:
             - name: empty-dir
               mountPath: /tmp
               subPath: tmp-dir
+            {{- if .Values.repoServer.configManagementPlugins.enabled }}
+            # The sidecars register their sockets to this volume for the repo server to find them.
+            - name: cmp-server-plugins
+              mountPath: /home/argocd/cmp-server/plugins
+            {{- end }}
           {{- if .Values.repoServer.extraVolumeMounts }}
           {{- include "common.tplvalues.render" (dict "value" .Values.repoServer.extraVolumeMounts "context" $) | nindent 12 }}
           {{- end }}
+        {{- if .Values.repoServer.configManagementPlugins.enabled }}
+        {{- range $plugin := .Values.repoServer.configManagementPlugins.plugins }}
+        {{- $additionalBinariesDir := $plugin.additionalBinariesDir | default "/usr/local/bin" | trimSuffix "/" }}
+        - name: {{ $plugin.name }}
+          image: {{ include "common.images.image" (dict "imageRoot" $plugin.sidecar.image "global" $.Values.global) }}
+          imagePullPolicy: {{ $plugin.sidecar.image.pullPolicy | default $.Values.image.pullPolicy }}
+          {{- if $plugin.sidecar.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" $plugin.sidecar.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          {{- $cmpSidecarSecurityContext := include "argocd.tplvalues.merge-with-precedence" (dict "values" (list $.Values.repoServer.containerSecurityContext $plugin.sidecar.containerSecurityContext) "context" $) | fromYaml }}
+          {{- if $cmpSidecarSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" $cmpSidecarSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if $plugin.sidecar.customCommand }}
+          command: {{- include "common.tplvalues.render" (dict "value" $plugin.sidecar.customCommand "context" $) | nindent 12 }}
+          {{- else }}
+          command:
+            - {{ $additionalBinariesDir }}/argocd-cmp-server
+          {{- end }}
+          {{- if $plugin.sidecar.customArgs }}
+          args: {{- include "common.tplvalues.render" (dict "value" $plugin.sidecar.customArgs "context" $) | nindent 12 }}
+          {{- end }}
```

Comment:
We always use `xxx.command` and `xxx.args` as paramaters name for users to customize their custom command/args. For consistency with the catalog, please use this naming.